samples/client-sdk/getting-started-daily-js/index.js

```diff
@@ -327,7 +404,6 @@ class DailyCallManager {
   toggleCamera() {
     this.call.setLocalVideo(this.isCameraMuted);
```

Comment:
if we get rid of our muted states, this needs to be updated to use `this.call.localVideo()`. Otherwise, right now the muted flags are subject to getting out of sync with reality. Let's say the user joins with mic/cam on and a little ways into the call, their headphones die. This demo isn't updating the flag when the track states changes so `isCameraMuted` remain `false`, while `trackInfo.state === 'playable'` is also `false` and the camera/mic state elements say 'Off'. Maybe not a huge deal since the buttons just say "toggle", but if someone were to inspect the values, it'd be confusing. To fix this, I think we just need to add a line to `updateUiForDevicesState()` so that this flag just always get updated to match the underlying state.